ui/apps/platform/src/Containers/MainPage/OutdatedVersionBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import { useSelector } from 'react-redux';
+import { Banner, Button } from '@patternfly/react-core';
+
+import { selectors } from 'reducers';
+
+function reloadWindow() {
+    window.location.reload();
+}
+
+function OutdatedVersionBanner(): ReactElement | null {
+    const isOutdatedVersion = useSelector(selectors.isOutdatedVersionSelector);
+
+    if (isOutdatedVersion) {
+        return (
+            <Banner className="pf-u-text-align-center" variant="warning">
+                It looks like this page is out of date and may not behave properly. Please{' '}
```

Comment:
Curious if you intentionally chose the curly bracket space `{' '}` here over a non-breaking space `&nbsp;`? Or is that just preference?

Reply:
Good eyes. This is JSX notation for trailing space when line breaks. From prettier, I think.